jsapp/jest/unit.config.ts

```diff
@@ -0,0 +1,60 @@
+import type {Config} from 'jest';
+import {defaults} from 'jest-config';
+
+// Config to run ☕ unit tests using the Jest runner
+//
+// To run the unit tests: 🏃
+//
+//     npx jest --config ./jsapp/jest/unit.config.ts
+//
+
+const config: Config = {
+
+  // Naming convention (*.tests.*)
+  testMatch: [ '**/?(*.)+(tests).(js|jsx|ts|tsx|es6|coffee)' ],
```

Comment:
No irregular whitespace:
```suggestion
  testMatch: ['**/?(*.)+(tests).(js|jsx|ts|tsx|es6|coffee)'],
```